setup.py

```diff
@@ -22,6 +22,13 @@ def read(*filenames, **kwargs):
     exec(file.read(), about)
 
```

Comment:
You do. It is designed for the case where a plugin wants to *safely* get standard in from old versions of Flake8 as well as new versions of Flake8.